docs/tutorials/fundamentals/tutorial-token-transfer-and-nft-purchase.md

```diff
@@ -88,7 +88,7 @@ Begin by checking the Factory Explorer in the Uniq Explorer. Confirm the on-chai
 
 ### Step 2: Initiate Purchase Action
 
-Navigate to the action builder in your toolkit and select the `eosio.nft.ft` contract. Choose the purchase action to start the process of buying a Uniq.
+Navigate to the factory in your toolkit and select the `eosio.nft.ft::purchase.a` action.
```

Comment:
As Aram mentioned it is probably better to refer to actions in a simple way.
So maybe replace the text with:

Navigate to the `Factory` page under the `Uniq Actions` section of the toolkit and then start typing `purchase` into the search bar. Then select the `Purchase Uniqs directly` action.

And update the image to indicate the result after the search so that there is less stuff on screen [image](blob:https://ultraio.atlassian.net/207d8bfb-5254-4ac6-9a13-a80fcbd96e79#media-blob-url=true&id=9042a508-7c09-4c4c-b43d-2e230f9565b4&contextId=45471&collection=)